.gitattributes

```diff
@@ -0,0 +1,2 @@
+# *nix shell scripts always use LF (see .editorconfig)
```

Comment:
Why is this in here? I don't see an .sh file being added.

Reply:
Generally makes sense to use LF for something that is intended for Linux systems, so let's keep it for now.